etcdserver/server.go

```diff
@@ -565,6 +565,8 @@ func (s *EtcdServer) RaftHandler() http.Handler { return s.r.transport.Handler()
 
 func (s *EtcdServer) Lessor() lease.Lessor { return s.lessor }
 
+func (s *EtcdServer) ApplyWait() wait.WaitTime { return s.applyWait }
```

Comment:
this shouldn't be exported to the outside world if possible, it's an internal implementation detail. Try
```go
func (s* EtcdServer) ApplyNotify() <-chan {
    return s.applyWait.Wait(atomic.LoadUint64(&s.committedIndex)) }
}
```